src/test/java/org/folio/rest/impl/FeeFineReportsAPITest.java

```diff
@@ -558,6 +581,18 @@ private Feefineaction createAction(String userId, int actionCounter, Account acc
     return action;
   }
 
+  private Feefineaction createActionWithNullComments(String userId, Account account, String dateTime,
```

Comment:
Do we really need two `createActionWithNullComments` methods? You used only one of them in test...